### DISCUSSION TITLE
Diapers change amount report

### DISCUSSION THREAD

**Discussion by maxdd** [babybuddy/babybuddy#798] [Q&A]
Hello,
For whatever reason I don't understand I cannot get a report on the number of diapers.
I'm using it since one month and it says that there isnt enough data for the report

**Comment by cdubz** [comment]
@maxdd which report specifically are you trying to look at that isn’t giving data? There are a couple for diaper changes.

**Reply by maxdd** [reply]
In the report list i think it is the one right after the BMI

**Reply by cdubz** [reply]
@maxdd that report is diaper change _amounts_ and will only have data to report on if you are using the "Amount" field on diaper changes (e.g., for weighing diapers). Are you using that field? If not you may be more interested in the diaper changes _types_ report that breaks down the number of changes by type.

**Reply by AlexKalopsia** [reply]
For what is worth, I had the same issue. I think the wording of "Amount" is quite confusing (ie "how many diapers" vs "how heavy it is"). In that sense I would go for "Content Amount" or "Weight".
